keydata.go

```diff
@@ -53,6 +56,92 @@ const (
 	AuthModePIN
 )
 
+type afSplitDataRawHdr struct {
+	Stripes uint32
+	HashAlg tpm2.HashAlgorithmId
+	Size    uint32
+}
+
+type afSplitDataRaw struct {
```

Comment:
this needs a doc comment and an explanation of what is going on (also it should probably been its own PR)

Reply:
I've added a comment for this now.